tests/foreman/virtwho/ui/test_hyperv.py

```diff
@@ -60,28 +71,25 @@ def test_positive_deploy_configure_by_id(self, default_org, session, form_data):
 
         :CaseImportance: High
         """
-        name = gen_string('alpha')
-        form_data['name'] = name
-        with session:
-            session.virtwho_configure.create(form_data)
-            values = session.virtwho_configure.read(name)
-            command = values['deploy']['command']
-            hypervisor_name, guest_name = deploy_configure_by_command(
-                command, form_data['hypervisor_type'], debug=True, org=default_org.label
-            )
-            assert session.virtwho_configure.search(name)[0]['Status'] == 'ok'
-            hypervisor_display_name = session.contenthost.search(hypervisor_name)[0]['Name']
-            vdc_physical = f'product_id = {settings.virtwho.sku.vdc_physical} and type=NORMAL'
-            vdc_virtual = f'product_id = {settings.virtwho.sku.vdc_physical} and type=STACK_DERIVED'
-            session.contenthost.add_subscription(hypervisor_display_name, vdc_physical)
-            assert session.contenthost.search(hypervisor_name)[0]['Subscription Status'] == 'green'
-            session.contenthost.add_subscription(guest_name, vdc_virtual)
-            assert session.contenthost.search(guest_name)[0]['Subscription Status'] == 'green'
-            session.virtwho_configure.delete(name)
-            assert not session.virtwho_configure.search(name)
+        name = form_data['name']
+        values = session.virtwho_configure.read(name)
+        command = values['deploy']['command']
+        hypervisor_name, guest_name = deploy_configure_by_command(
```

Comment:
Not sure, if UI tests would work the same way without a session context manager (I think we shouldn't use fixtures here for creating and deleting) , never tried!
@omkarkhatavkar @jyejare ^^

Reply:
@JacobCallahan Could you please help to review and merge the PR? Thank you very much

Reply:
@yanpliu what was the reason to remove the `with session` context manager, This would likely not run the __exit__ method and cause the browser to be stayed up for a little long time.

Reply:
Ohh Okay `virtwho_config` fixture handles the session